cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -1336,7 +1408,7 @@ features:
         name: Deploy
       one_step_deploy_plugin:
         name: Deploy
-    connectErrorMessage: Cannot connect to Market
+    connectErrorMessage: Cannot connect to Hub
```

Comment:
in this case, should `Hub` be lower case?

Reply:
Nope, since 'Hub' here  is a proper noun